optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -929,6 +942,8 @@ def __init__(
         optuna_callbacks: Optional[List[Callable[[Study, FrozenTrial], None]]] = None,
         verbosity: Optional[int] = None,
         show_progress_bar: bool = True,
+        model_dir: Optional[str] = None,
+        return_cvboost: Optional[bool] = None,
```

Comment:
The argument name seems to be `return_cvbooster` according to the [reference](https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.cv.html).
```suggestion
        return_cvbooster: Optional[bool] = None,
```